internal/lib/sandbox/sandbox.go

```diff
@@ -140,6 +140,18 @@ func (s *Sandbox) IPs() []string {
 	return s.ips
 }
 
+// FirstIP returns either the first result of `IPs` or the host IP if the
+// sandbox is running on host network
+func (s *Sandbox) FirstIP() string {
+	// Use the first IP of the sandbox as target if non host-network
+	if len(s.IPs()) > 0 {
+		return s.IPs()[0]
+	}
+
+	// For host networks
+	return "localhost"
```

Comment:
I suggest to not return localhost to avoid the resolution problem . it may resolve to ::1 or to 127.0.0.1 , but the application may not be bound to the same address
xref. https://gist.github.com/aojea/94f6f483173641647c731f582e52f0b0

@danwinship what do you think? is forcing to `127.0.0.1` safe adding a comment explaining or a TODO?

Reply:
Thanks, changed it to `127.0.0.1` for now.

Reply:
rereading my comment it may suggest something different, I meant to keep the FirstIP code but instead of `return localhost` as the default just `return 127.0.0.1`